journal/src/test/java/io/camunda/zeebe/journal/file/SegmentedJournalTest.java

```diff
@@ -661,7 +662,12 @@ void shouldNotPreallocateSegmentFiles(final @TempDir Path tmpDir) {
     }
 
     // then
-    PosixPathAssert.assertThat(firstSegment).hasRealSizeLessThan(segmentSize);
+    if (Platform.IS_MAC) {
+      // on MAC_OS the memory gets pre-allocated by SegmentAllocator. See issue: #12664
```

Comment:
The error message that we get is that the entirety of the memory is being allocated. Is it possible that mac os has different definitions for allocated? since the `stat()` method is returning a different result?

Reply:
Besides this https://developer.apple.com/documentation/foundation/file_system/about_apple_file_system#2990507 could not find much information about the subject. 
But I also changed the comment there.